image/image_linux.go

```diff
@@ -400,13 +402,25 @@ func (s *imageSeeder) setModesDirs() error {
 	return nil
 }
 
-func (s *imageSeeder) start(db *asserts.Database, optSnaps []*seedwriter.OptionsSnap) error {
+func (s *imageSeeder) start(optSnaps []*seedwriter.OptionsSnap) error {
 	if err := s.w.SetOptionsSnaps(optSnaps); err != nil {
 		return err
 	}
+
+	// TODO: developer database in home or use snapd (but need
```

Comment:
~~(nitpick) this seems to be a bit hard to read. should this be: `TODO: *use* developer database ...` ?~~ 
I see this is just moved code so please ignore me